src/tests/server.rs

```diff
@@ -234,7 +332,7 @@ async fn failed_stop_indexer(#[future] setup_server: SocketAddr) {
     );
 
     // sleep for 100ms to let the indexer stop.
-    tokio::time::sleep(Duration::from_millis(100)).await;
+    tokio::time::sleep(Duration::from_millis(500)).await;
```

Comment:
update comment

Reply:
I am actually removing this. There is some uncertainity in the tests right now, this tests passes mostly but fails sometimes because the status is Stopped instead of FailedStopping. Created an issue for this #37